sdk/core/core-https/src/nodeHttpsClient.ts

```diff
@@ -198,9 +198,11 @@ class NodeHttpsClient implements HttpsClient {
     const proxySettings = request.proxySettings;
     if (proxySettings) {
       if (!this.proxyAgent) {
+        const parsedUrl = new URL(proxySettings.host);
```

Comment:
this can throw on invalid URLs, so do we need to worry about catching that/logging or something? Or just let it bubble up?

Reply:
Maybe we could throw with a better message stating the expected format.  A question related to `ProxySettings`, should we align it with the new underlying options? ie, adding `protocol`, and make `host` just the `hostname` part.

Reply:
actually it would  be replacing the old `host` with `protocol` and `hostname`

Reply:
That sounds good to me, as long as the environment variables still work as expected

Reply:
On second thought, I don't want to break from core-http as this is not important enough to do. so will keep the old `ProxySettings`.

Reply:
Changed to catch and throw another error with better message instead.